source/_patterns/04-pages/00-typography.mustache

```diff
@@ -1,11 +1,11 @@
 <main role="main" class="container">
   <h2>Headlines</h2>
 
-  {{> atoms-h1(content: 'This is a simple <H1> headline') }}
-  {{> atoms-h2(content: 'This is a simple <H2> headline') }}
-  {{> atoms-h3(content: 'This is a simple <H3> headline') }}
-  {{> atoms-h4(content: 'This is a simple <H4> headline') }}
-  {{> atoms-h5(content: 'This is a simple <H5> headline') }}
+  {{> atoms-h1(h1Content: 'This is a simple <H1> headline') }}
```

Comment:
Good, this variable name change avoids confusion on other files.

Reply:
They are scoped as normal Javascript variables. So with this knowledge they don't overwrite other "content", but you are right. I changed it because if readability, but also when you start using more than one "content" in one document, the document doesn't know for which "content" it is supposed to be. Haha. :)

Reply:
Yes! Exactly